### PR TITLE
Ginny/fhdds/front end UI upgrade base

### PR DESCRIPTION
**Pull request by ginnyamazed**
create new layout template and ViewHelpers- redirect existing pages to main_template_old and OldViewHelpers.
Keep page scripts in page setup.
Update play-frontend-hmrc to 3.22.0 (latest version)

### CODE REVIEW

File: app/uk/gov/hmrc/fhregistrationfrontend/views/forms/company_officers.scala.html [hunk 1]
```diff
@@ -23,7 +23,7 @@
 @import uk.gov.hmrc.fhregistrationfrontend.config.AppConfig
 
 @this(
-        main_template: main_template
+        main_template_old: main_template_old
```
Comment (priyankadhiman3006) [inline app/uk/gov/hmrc/fhregistrationfrontend/views/forms/company_officers.scala.html:26]: no need to change  main_template to main_template_old here keep it `main_template:main_template_old`

File: app/uk/gov/hmrc/fhregistrationfrontend/views/forms/company_officers.scala.html [hunk 2]
```diff
@@ -71,7 +71,7 @@
   Messages("fh.company_officers.title")
 }
 
-@main_template(title = titlePrefix + title, scriptElem = Some(pageScripts)) {
+@main_template_old(title = titlePrefix + title, scriptElem = Some(pageScripts)) {
```
Comment (priyankadhiman3006) [inline app/uk/gov/hmrc/fhregistrationfrontend/views/forms/company_officers.scala.html:74]: then this change is not needed

File: app/uk/gov/hmrc/fhregistrationfrontend/views/forms/examples.scala.html [hunk 3]
```diff
@@ -32,7 +32,7 @@
 
 @stylesheets = {<link rel="stylesheet" href='@routes.Assets.at("stylesheets/fh.css")'/>}
 @scripts = {<script type="text/javascript" src='@routes.Assets.at("javascripts/fh.js")'></script><script type="text/javascript" src='@routes.Assets.at("javascripts/addressLookup.js")'></script>}
-@main_template(linkElement = Some(stylesheets), pageScripts = Some(scripts), mainClass = Some("full-width")) {
+@main_template_old(linkElement = Some(stylesheets), pageScripts = Some(scripts), mainClass = Some("full-width")) {
```
Comment (priyankadhiman3006) [inline app/uk/gov/hmrc/fhregistrationfrontend/views/forms/examples.scala.html:35]: same here

File: conf/messages [hunk 4]
```diff
@@ -1,4 +1,6 @@
 #COMMON
+service.name = Fulfilment House Due Diligence Scheme
```
Comment (priyankadhiman3006) [inline conf/messages:2]: I think service name should be `Apply for the Fulfilment House Due Diligence Scheme`. what do you think?

File: app/uk/gov/hmrc/fhregistrationfrontend/views/forms/company_officers.scala.html (hunk 1 above)
Comment (ginnyamazed) [reply app/uk/gov/hmrc/fhregistrationfrontend/views/forms/company_officers.scala.html:26]: Fixed, not sure when that page changed

File: app/uk/gov/hmrc/fhregistrationfrontend/views/forms/examples.scala.html [hunk 5]
```diff
@@ -14,14 +14,14 @@
...
-@import uk.gov.hmrc.fhregistrationfrontend.views.html.main_template
+@import uk.gov.hmrc.fhregistrationfrontend.views.html.main_template_old
 @import uk.gov.hmrc.fhregistrationfrontend.views.helpers._
 @import uk.gov.hmrc.fhregistrationfrontend.views.html.helpers._
 @import uk.gov.hmrc.fhregistrationfrontend.forms.navigation.Navigation
 @import uk.gov.hmrc.fhregistrationfrontend.config.AppConfig
 
 @this(
-        main_template: main_template
+        main_template_old: main_template_old
```
[the remark this answers is not in the document]
Comment (ginnyamazed) [reply app/uk/gov/hmrc/fhregistrationfrontend/views/forms/examples.scala.html:24]: fixed

File: app/uk/gov/hmrc/fhregistrationfrontend/views/forms/examples.scala.html (hunk 3 above)
Comment (ginnyamazed) [reply app/uk/gov/hmrc/fhregistrationfrontend/views/forms/examples.scala.html:35]: fixed

File: conf/messages (hunk 4 above)
Comment (ginnyamazed) [reply conf/messages:2]: https://github.com/hmrc/play-frontend-hmrc#getting-started

Comment (priyankadhiman3006) [reply conf/messages:2]: If you look at line no 4 in this file it is 

`app.name=Apply for the Fulfilment House Due Diligence Scheme`
Plus if you look into the service , the name in the header is Apply for the Fulfilment House Due Diligence Scheme

File: app/uk/gov/hmrc/fhregistrationfrontend/views/forms/company_officers.scala.html (hunk 1 above)
Comment (priyankadhiman3006) [reply app/uk/gov/hmrc/fhregistrationfrontend/views/forms/company_officers.scala.html:26]: I can still see this as main_template_old: main_template_old, have the changes been pushed?

File: app/uk/gov/hmrc/fhregistrationfrontend/views/forms/company_officers.scala.html (hunk 2 above)
Comment (priyankadhiman3006) [reply app/uk/gov/hmrc/fhregistrationfrontend/views/forms/company_officers.scala.html:74]: This is still not done

**Review by daglisood — APPROVED** [review]

**Pull request closed without merging**